src/VisualStudio/Core/Impl/Options/GridOptionPreviewControl.xaml

```diff
@@ -87,23 +87,25 @@
                 </GroupStyle>
             </DataGrid.GroupStyle>
             <DataGrid.Columns>
-                <DataGridTextColumn
+                <DataGridTemplateColumn
                     x:Name="description" 
                     Header="{x:Static local:GridOptionPreviewControl.DescriptionHeader}"
-                    Binding="{Binding Description, Mode=OneWay}" 
                     Width="4.5*"
                     IsReadOnly="True">
-                    <DataGridTextColumn.ElementStyle>
-                        <Style TargetType="TextBlock">
-                            <Setter Property="Margin" Value="{Binding DescriptionMargin, Converter={StaticResource MarginConverter}}" />
-                            <Setter Property="Padding" Value="{StaticResource ResourceKey=cellPadding}" />
-                            <Setter Property="VerticalAlignment" Value="Center"/>
-                            <Setter Property="HorizontalAlignment" Value="Left" />
-                            <Setter Property="TextWrapping" Value="Wrap" />
-                            <Setter Property="AutomationProperties.Name" Value="{Binding GroupName}" />
-                        </Style>
-                    </DataGridTextColumn.ElementStyle>
-                </DataGridTextColumn>
+                    <DataGridTemplateColumn.CellTemplate>
+                        <DataTemplate>
+                            <local:TextBlockWithDataItemControlType
```

Comment:
Left everything the same as the "Setter" elements before, with the exceptions of "Text", "AutomationProperties.Name", and "Focusable".